stable/atlantis/templates/_helpers.tpl

```diff
@@ -40,4 +40,22 @@ Create the name of the service account to use
 {{- else -}}
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{/*
+Defines the internal kubernetes address to Atlantis
+*/}}
+{{- define "atlantis.http.scheme" -}}
+{{- if .Values.tlsSecretName -}}
+http
+{{- else -}}
+https
+{{- end -}}
+{{- end -}}
+
+{{/*
+Defines the internal kubernetes address to Atlantis
+*/}}
+{{- define "atlantis.url" -}}
```

Comment:
should this be `atlantis.internal-url` to make it clear that it's the cluster internal URL, not the external URL or anything else?

Reply:
No strong feelings, sounds good.